src/openrct2/paint/Supports.h

```diff
@@ -94,6 +94,19 @@ enum class MetalSupportType : uint8_t
     BoxedCoated,
 };
 
+enum class MetalSupportPlace : uint8_t
+{
+    TopCorner = 0,
+    LeftCorner = 1,
+    RightCorner = 2,
+    BottomCorner = 3,
+    Centre = 4,
+    TopLeftSide = 5,
+    TopRightSide = 6,
+    BottomLeftSide = 7,
+    BottomRightSide = 8,
```

Comment:
I'm having a hard time visualizing this, can you draw them? To me a corner has to be defined by two dimensions

Reply:
If you look at the RCT grid, every tile is rotated 45 degrees, so you have the corners at the top, bottom, left and right of the screen:

![afbeelding](https://github.com/OpenRCT2/OpenRCT2/assets/1478678/035aa42a-aefe-4c4d-b33c-ac7af9eb7cfe)